<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassAnnotation</class>
 <widget class="qMRMLWidget" name="ClassAnnotation">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-54</y>
        <width>759</width>
        <height>646</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="scrollLayout">
       <item>
        <widget class="QPushButton" name="loadButton">
         <property name="text">
          <string>Select Dataset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="classificationTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="classificationGroupBox">
         <property name="title">
          <string>Classification</string>
         </property>
         <layout class="QGridLayout" name="classificationLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="class0Button">
            <property name="styleSheet">
             <string>background-color: #FF4C4C; color: black;</string>
            </property>
            <property name="text">
             <string>Class 0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLCDNumber" name="lcdClass0"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="class1Button">
            <property name="styleSheet">
             <string>background-color: #4CAF50; color: black;</string>
            </property>
            <property name="text">
             <string>Class 1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="lcdClass1"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="class2Button">
            <property name="styleSheet">
             <string>background-color: #FF9800; color: black;</string>
            </property>
            <property name="text">
             <string>Class 2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLCDNumber" name="lcdClass2"/>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="class3Button">
            <property name="styleSheet">
             <string>background-color: #FFD700; color: black;</string>
            </property>
            <property name="text">
             <string>Class 3</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLCDNumber" name="lcdClass3"/>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="class4Button">
            <property name="styleSheet">
             <string>background-color: #2196F3; color: black;</string>
            </property>
            <property name="text">
             <string>Class 4</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLCDNumber" name="lcdClass4"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="reviewGroupBox">
         <property name="title">
          <string>Review</string>
         </property>
         <layout class="QVBoxLayout" name="reviewLayout">
          <item>
           <widget class="QGroupBox" name="automaticReviewGroupBox">
            <property name="title">
             <string>Automatic Review</string>
            </property>
            <layout class="QVBoxLayout" name="automaticReviewLayout">
             <item>
              <widget class="QLabel" name="label_cases">
               <property name="text">
                <string>Choose the number of cases per class</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="casesInput">
               <property name="placeholderText">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Random Check</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="nextPatientButton">
               <property name="text">
                <string>Next Patient</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="manualReviewGroupBox">
            <property name="title">
             <string>Manual Review</string>
            </property>
            <layout class="QGridLayout" name="manualReviewLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_selectPatient">
               <property name="text">
                <string>Select a patient to review</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="patientDropdown"/>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QPushButton" name="reviewButton">
               <property name="text">
                <string>Review</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qmrmlwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
